static/js/containers/ProfileImage_test.js

```diff
@@ -131,11 +129,14 @@ describe("ProfileImage", () => {
     [PROFILE_IMAGE_SMALL, true],
     [PROFILE_IMAGE_MEDIUM, false]
   ].forEach(([imageSize, exp]) => {
-    it(`should call makeProfileImageUrl with ${String(
-      exp
-    )} if imageSize is ${imageSize}`, () => {
-      renderProfileImage({ imageSize })
-      assert.ok(makeProfileImageUrlStub.calledWith(thatProfile, exp))
+    it(`image url should ${
+      exp ? "" : " not"
+    } be small if imageSize is ${imageSize}`, () => {
```

Comment:
Optional change, you can take or leave: The `shouldIf` helper func can be used here. 

`image url ${shouldIf(exp)} be small if imageSize is ${imageSize}`